samples/ClassroomLabs/Scripts/GenericPreparation/Prepare-MicrosoftStoreApplications.ps1

```diff
@@ -0,0 +1,133 @@
+<#
+SYNOPSIS
+This script prepares computer for class by aiding in the deletion of unneeded Microsoft Store applications.  Remaining Microsoft Store applications are updated.
+#>
+
+[CmdletBinding()]
+param( )
+
+###################################################################################################
+#
+# PowerShell configurations
+#
+
+# NOTE: Because the $ErrorActionPreference is "Stop", this script will stop on first failure.
+#       This is necessary to ensure we capture errors inside the try-catch-finally block.
+$ErrorActionPreference = "Stop"
+
+# Hide any progress bars, due to downloads and installs of remote components.
+$ProgressPreference = "SilentlyContinue"
+
+# Ensure we set the working directory to that of the script.
+Push-Location $PSScriptRoot
+
+# Discard any collected errors from a previous execution.
+$Error.Clear()
+
+# Configure strict debugging.
+Set-PSDebug -Strict
+
+###################################################################################################
+#
+# Handle all errors in this script.
+#
+
+trap {
+    # NOTE: This trap will handle all errors. There should be no need to use a catch below in this
+    #       script, unless you want to ignore a specific error.
+    $message = $Error[0].Exception.Message
+    if ($message) {
+        Write-Host -Object "`nERROR: $message" -ForegroundColor Red
+    }
+
+    Write-Host "`nThe script failed to run.`n" -ForegroundColor Red
+
+    # IMPORTANT NOTE: Throwing a terminating error (using $ErrorActionPreference = "Stop") still
+    # returns exit code zero from the PowerShell script when using -File. The workaround is to
+    # NOT use -File when calling this script and leverage the try-catch-finally block and return
+    # a non-zero exit code from the catch block.
+    exit -1
+}
+
+###################################################################################################
+#
+# Functions used in this script.
+#             
+
+<#
+.SYNOPSIS
+Returns true if script is running with administrator privileges and false otherwise.
+#>
+function Get-RunningAsAdministrator {
+    [CmdletBinding()]
+    param()
+    
+    $isAdministrator = ([Security.Principal.WindowsPrincipal] [Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole([Security.Principal.WindowsBuiltInRole] "Administrator")
+    Write-Verbose "Running with Administrator privileges (t/f): $isAdministrator"

```

Comment:
Change to the following in your function to ensure you return 1 and only 1 value. You can use this technique to suppress multiple lines. The output will still be written but it will not be part of your function's return value.
``` PS1
$null = @(
    Write-Verbose "Running with Administrator privileges (t/f): $isAdministrator"
)
```